pkg/network/http/ebpf_ssl.go

```diff
@@ -327,3 +300,18 @@ func (o *sslProgram) httpsSupported() bool {
 
 	return kversion >= kernel.VersionCode(5, 5, 0)
 }
+
+func (o *sslProgram) sysOpenAt2Supported() bool {
+	ksymPath := filepath.Join(o.cfg.ProcRoot, "kallsyms")
+	missing, err := ddebpf.VerifyKernelFuncs(ksymPath, []string{doSysOpenAt2.function})
```

Comment:
```suggestion
	missing, err := ddebpf.VerifyKernelFuncs(ksymPath, []string{doSysOpenAt2.section})
```
`section` is what will match to the kernel function, right?

Reply:
yes good catch